core/src/main/java/org/mskcc/cbio/maf/MafUtil.java

```diff
@@ -361,6 +367,15 @@ public MafUtil(String headerLine) {
                 driverTiersIndex = i;
             } else if(header.equalsIgnoreCase(DRIVER_TIERS_FILTER_ANNOTATION)) {
                 driverTiersAnnIndex = i;
+            }  else if (namespaces != null && !namespaces.isEmpty()) {
+                for (String ns : namespaces) {
+                    if (header.toLowerCase().startsWith(ns + NAMESPACE_DELIMITER)) {
+                        String nsKey = header.toLowerCase().replace(ns + NAMESPACE_DELIMITER, "");
```

Comment:
If we are lower casing the key here we will need to update the constructor inside the AlleleSpecificCopyNumber model which gets the values out. It's currently hard-coded to match the test MAF - e.g TOTAL_COPY_NUMBER instead of total_copy_number